src/data/contributors/Rajnandini.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Rajnandini S T",
+  "skills": ["Python"],
+  "cause": "knowledge and contributing in opensource",
+  "image": "Avatar5",
+  "github": "https://github.com/rajnandini-tambavekar",
+  "twitter": "--",
+  "linkedin": "--"
```

Comment:
remove it or leave it blank if you don't have accounts.

Reply:
ok , should I create a new branch and new file or edit in same branch n file ?

Reply:
actually no need to create a new branch. You can edit that file and then commit like you have previously did.

Reply:
I think , I have done it .Will you please check it ?